### DISCUSSION TITLE
Support upgrading Go version in the go.mod

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#26865] [Request Help] [answered]
### Tell us more.

It would be great to support upgrading Go version in the go.mod, thanks!

For example

```go
module github.com/hongbo-miao/hongbomiao.com/api-go

go 1.20

require (
	github.com/gin-gonic/gin v1.9.1
)

// ...
```

It would be great to bump `go 1.20` to `go 1.21`. Thanks!

**Comment by viceice** [comment, marked as answer]
this should be possible but is disabled by default. 

https://github.com/renovatebot/renovate/blob/274aecdb9601a0d48b548a7bf4942220a5cb6c14/lib/modules/manager/gomod/extract.ts#L49

https://docs.renovatebot.com/modules/versioning/#go-modules-directive-versioning

**Reply by rarkins** [reply]
I think that it's not disabled, but set to "replace", which means it never needs replacing. I think it updates if you set to rangeStrategy=bump
